### DISCUSSION TITLE
shuffle button when inside a folder

### DISCUSSION THREAD

**Discussion by danielb2** [namidaco/namida#79] [Ideas]
currently I have to excit one level up, and long-press the folder to get access to the shuffle button. It's a  bit unintutive to navigate back out of the folder after looking at the songs just to shuffle them

**Comment by MSOB7YY** [comment]
you can enable "floating action button" from extra settings, this will show stuffle button whereas possible 

i don't think it's quite possible for folder tracks since they have quite simple design

**Comment by danielb2** [comment]
oh that's awesome. thanks for that. problem solved :)
